eu.nokun.MirrorHall.json

```diff
@@ -0,0 +1,104 @@
+{
+    "app-id": "eu.nokun.MirrorHall",
```

Comment:
Can you add a link to the gitlab of this project here https://nokun.eu/projects ?

Reply:
I am writing a post about this on notes.nokun.eu, is that also fine?